usecase/usecase-wrt-android-tests/samples/UpgradeLib/res/testapp/ssbl/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "example_ssbl",
+  "start_url": "index.html",
+  "xwalk_app_version": "1.0.1",
+  "xwalk_package_id": "org.xwalk.example"
```

Comment:
"org.xwalk.example" is not a good package id since it is to simple and may be same with others.